components/datetime/Cargo.toml

```diff
@@ -64,15 +64,15 @@ bench = false  # This option is required for Benchmark CI
 
 [features]
 std = ["icu_provider/std", "icu_locid/std", "icu_calendar/std"]
-default = ["serialize"]
+default = ["serde"]
```

Comment:
We also decided that this will not be a default feature, maybe clean these up now as well?